core/lib/da_client/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "zksync_da_client"
+version.workspace = true
+edition.workspace = true
+authors.workspace = true
+homepage.workspace = true
+repository.workspace = true
+license.workspace = true
+keywords.workspace = true
+categories.workspace = true
+
+[dependencies]
+serde = { workspace = true, features = ["derive"] }
+tracing = "0.1.40"
+async-trait = "0.1.74"
```

Comment:
Please import these dependencies from the workspace. Ditto for `zksync_default_da_clients`.